src/components/AddBook.js

```diff
@@ -1,14 +1,36 @@
 import '../style/AddBook.css';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { useState } from 'react';
+import { addBook } from '../redux/books/books';
 
 function Books() {
+  const dispatch = useDispatch();
+  const newbook = {
+    id: '',
+    title: '',
+    author: '',
+  };
+  const [mybook, setMyBook] = useState(newbook);
+  const addNew = () => {
+    dispatch(addBook(mybook));
+  };
```

Comment:
- [ ] **[OPTIONAL]** I think it would be nice to clear the inputs after dispatching the book.